contexts/credentials/v2

```diff
@@ -193,6 +193,23 @@
       }
     },
 
+    "ShaclSchemaCredential": "https://w3.org/2018/credentials#ShaclSchemaCredential",
```

Comment:
I believe this should be added to the [VC Specs Directory](https://w3c.github.io/vc-specs-dir/#credential-schema) before merging this PR to provide additional context on how one would go about using a SHACL schema.

Reply:
Additionally, in this form this means extending the official VCDM vocabulary.

Alternatively, a separate URL is used for this property to reflect that this is an extension to the core (if this is the way we go).

Reply:
Thanks for pointing me to the VC Specs Directory, @decentralgabe. What's the most reasonable way to get this done?
I noticed https://github.com/w3c/vc-specs-dir#adding-new-entries-to-the-directory.

All entries [in this table](https://w3c.github.io/vc-specs-dir/#credential-schema) point to proper specification documents. So, would I have to write such a specification document to explain how a SHACL shapes graph (to use the proper term) should be used to validate a credential? So far, I would have considered that a straightforward application of SHACL's "validation of a focus node \[i.e., the credentialSubject\] against a shape \[for all applicable shapes\]" ([as specified here](https://www.w3.org/TR/shacl/#validation-definition)) to be explained by a few bullet points, but given @iherman's additional comments, maybe that's not the case.

Reply:
A specification that's useful for implementers would be ideal. If some such documents already exist maybe they are fine to point to.